packages/react-core/src/components/Avatar/examples/Avatar.md

```diff
@@ -15,5 +15,11 @@ import React from 'react';
 import { Avatar } from '@patternfly/react-core';
 import avatarImg from './examples/avatarImg.svg';
 
-<Avatar src={avatarImg} alt="avatar"></Avatar>
+class AvatarExample extends React.Component {
```

Comment:
made this change because i noticed the codesandbox wasn't working before. should probably go through and update all the examples that use the previous syntax @redallen ?

Reply:
All you need is to name the example like this @jschuler :
```jsx
AvatarExample = () => <Avatar src={avatarImg} alt="avatar"></Avatar>
```

Reply:
codesandbox would work (after adding `const` in front), but our docs would not
```
SyntaxError: Unexpected token (1:8)
1 : return (const AvatarExample = () => <Avatar src={avatarImg} alt="avatar"></Avatar>)
```